README.md

```diff
@@ -1,6 +1,141 @@
+[![Go Reference](https://pkg.go.dev/badge/foxygo.at/evy.svg)](https://pkg.go.dev/foxygo.at/evy)
 
+Evy is a simple programming language, made to learn coding.
+
+Evy has a minimalist syntax with fewer special characters and advanced
+concepts than most programming languages. It also has a small set of
+[built-in functions](docs/builtins.md) that are easy to understand and
+remember, but it still is powerful enough for user interaction, games,
+and animations. Evy bridges the gap between block-based languages like
+[Scratch] and conventional languages like Python or JavaScript.
+
+[Scratch]: https://scratch.mit.edu/
+
+## 🌱 Getting Started
+
+You can try Evy online at [evy.dev] and browse the examples.
+Alternatively you can [install](#-installation) the `evy` toolchain
+locally.
+
+Here is a "hello world" program in Evy
+
+    print "Hello World!"
+
+<details>
+  <summary>Screencast: How to code a simple animation in Evy.</summary>
+
+[![Coding evy](docs/img/purple-dot.gif)](https://evy.dev)
+
+[Animation source code]
+
+</details>
+
+[evy.dev]: https://evy.dev
+[Animation source code]: https://evy.dev/#content=H4sIAAAAAAAAEzWLwQqAIBBE7/sVg/fSiC6BHyO2B0FXWazvz4qGGXjMMKVejM09thaRpbNSrLkqTDu1ZTak2D1WRzFpzAwlqoIgqYTOhKHRBn1J4UcmuHn5lv/CctAN/HT8mWwAAAA=
+
+## 📖 Documentation
+
+Here are some resources for learning more about Evy:
+
+- [Learn and Practise](https://github.com/foxygoat/evy/wiki): A guided tour that teaches you programming using Evy.
+- [Syntax by Example](docs/syntax_by_example.md): A collection of examples that illustrate the Evy syntax.
+- [Built-in Documentation](docs/builtins.md): Details on built-in functions and events in Evy.
+- [Language Specification](docs/spec.md): A formal definition of the Evy syntax and language.
+
+## 📦 Installation
+
+You can install the Evy toolchain locally to run it from your command
+line. To learn how to use the Evy toolchain, read the
+[usage documentation](docs/usage.md) or run `evy --help`.
+
+### Linux and Windows
+
+Download the [latest release] for your platform, unzip it and add `evy`
+to your path.
+
+### macOS
+
+Use [Homebrew] to install `evy`.
+
+    brew install foxygoat/tap/evy
+
+[latest release]: https://github.com/foxygoat/evy/releases/latest
+[Homebrew]: https://brew.sh/
+
+## 💻 Development
+
+The Evy compiler is written in [Go] and built using the Go and
+[TinyGo] compilers. TinyGo targets [WebAssembly], which allows Evy
+source code to be compiled and run in a web browser. The browser
+runtime is written in plain JavaScript without the use of frameworks.
+
+To build the Evy source code, [clone] this repository and
+[activate Hermit] in your terminal. Then, build the sources with
+
+    make all
+
+You can list make targets, execute a full CI run locally or serve the
+frontend locally with
+
+    make help
+    make ci
+    make frontend-serve
+
+<details>
+  <summary>Activated Hermit auto-installs pinned tools.</summary>
```

Comment:
Sorry I meant to comment on this before. I find this sentence hard to understand and I don't think it will make much sense to many people.

Perhaps something like: "Automatic tool installation with Hermit". I don't know if that's in line with what you wanted though.

Reply:
I'd like it to be summary, ie full sentence. so maybe:

Hermit automatically installs tools.

I've changed it to that.....